crates/top/rerun/src/commands/rrd/merge_compact.rs

```diff
@@ -87,6 +89,14 @@ pub struct CompactCommand {
     #[arg(long = "max-rows-if-unsorted")]
     max_rows_if_unsorted: Option<u64>,
 
+    /// Configures the number of extra compaction passes to run on the data.
+    ///
+    /// The more passes, the better the compaction quality.
+    ///
+    /// If the data reaches optimal compaction, the computation will stop immediately in any case.
+    #[arg(long = "num-pass", default_value_t = 10)]
+    num_passes: u32,
```

Comment:
```suggestion
    num_extra_passes: u32,
```

what threw me off here at first is that we are always running at least one pass by simply reading the data in